snc.sh

```diff
@@ -169,6 +169,31 @@ export OPENSHIFT_INSTALL_INVOKER="codeReadyContainers"
 
 ${OPENSHIFT_INSTALL} --dir $INSTALL_DIR create cluster --log-level debug
 
+# export the kubeconfig
+export KUBECONFIG=$INSTALL_DIR/auth/kubeconfig
+
+# Add kubelet-bootstrap-cred-manager daemonset
+oc apply -f kubelet-bootstrap-cred-manager-ds.yaml
+
+# Delete the current csr signer to get new request. 
+oc delete secrets/csr-signer-signer secrets/csr-signer -n openshift-kube-controller-manager-operator
+
+# Wait for 5 min to make sure cluster is stable again.
+sleep 300
+
+# Remove the 24 hours certs and bootstrap kubeconfig
+# This kubeconfig again get generated and new certs will be created in pki folder
+# which will have 30 days validity.
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo rm -fr /var/lib/kubelet/pki
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo rm -fr /var/lib/kubelet/kubeconfig
```

Comment:
Removing these does not seem to be part of the instructions you linked to? https://blog.openshift.com/enabling-openshift-4-clusters-to-stop-and-resume-cluster-vms/
Is this mandatory?

Reply:
@cfergeau Without this we need to reboot the cluster to get the new csr atleast the node one so from crc prospective this is mandatory.

Reply:
It seems unexpected though https://bugzilla.redhat.com/show_bug.cgi?id=1760181#c4
Maybe add a comment that this is a workaround?